src/components/ui/SelectionDrawer/SelectionDrawerRadioGroup.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * This file is based on the Radio Group component from shadcn and customized for our needs.
+ *
+ * See the official docs for more info:
+ * shadcn/ui:https://ui.shadcn.com/docs/components/radio-group
+ * Radix UI: https://www.radix-ui.com/primitives/docs/components/radio-group
+ */
+import { cn } from '@/lib/tailwind/utils';
+
+import {
+  Indicator as PrimitiveIndicator,
+  Item as PrimitiveItem,
+  Root as PrimitiveRoot,
+} from '@radix-ui/react-radio-group';
+import { ComponentPropsWithoutRef, ElementRef, forwardRef } from 'react';
+
+const SelectionDrawerRadioGroup = forwardRef<
+  ElementRef<typeof PrimitiveRoot>,
+  ComponentPropsWithoutRef<typeof PrimitiveRoot>
+>(({ className, ...props }, ref) => {
+  return <PrimitiveRoot className={cn('', className)} {...props} ref={ref} />;
+});
+SelectionDrawerRadioGroup.displayName = PrimitiveRoot.displayName;
```

Comment:
<img width="1272" alt="image" src="https://github.com/genesis-tech-tribe/nishiki-frontend/assets/25422441/74b56a3d-1ffd-4d09-b3be-e47d8f721ed8">
In my environment, This lint error shows up. How about your environment?

> Unsafe member access .displayName on an `any` value.eslint[@typescript-eslint/no-unsafe-member-access](https://typescript-eslint.io/rules/no-unsafe-member-access)

Reply:
@kotaaaa 
No, I don't see the error in my environment.

Do you remember there was exactly the same error in my IDE on your PR?
https://github.com/genesis-tech-tribe/nishiki-frontend/pull/68#discussion_r1421693075

It could be caused by some misconfiguration in `.eslintrc.js`.

Reply:
After restart my vscode, error gone away. (I don't know why though) Thank you for checking!